### ISSUE TITLE
Update skew and kurtosis to return NaN when array is 100% duplicated

### ISSUE THREAD

**Issue by phobson** [dask/crick#33]
Crick was originally written to be compatible with scipy v1.7. 

Now with a more modern scipy (v.19), skew and kurtosis return NaN when `nunique(array) == 1`. Previously in Scipy and currently in crick, the kurtosis values of a single-unique-value array are either 0 when the `fisher` parameter is True or or 3 when it is False. The presence of the issue is demonstrated in the test suite & the case is currently marked as an `xfail`: https://github.com/dask/crick/blob/master/crick/tests/test_stats.py#L72

Basically, (in pure C), we need to be able to detect the number of unique elements in the tdigest array and set the skew and kurtosis functions to return NaN in those cases.

This issue can be considered a "part two" of #31.
